app/grants/views.py

```diff
@@ -1704,3 +1707,55 @@ def toggle_grant_favorite(request, grant_id):
     return JsonResponse({
         'action': 'follow'
     })
+
+
+@login_required
+def verify_grant(request, grant_id):
+    grants = Grant.objects.filter(pk=grant_id)
+
+    if not grants.exists():
+        return JsonResponse({
+            'ok': False,
+            'msg': 'Grant doesn\'t exists'
+        })
+
+    grant = grants.first()
```

Comment:
^ could we do a grants.object.get ? every grant is unique 
If we have duplicate entries -> something is wrong

Reply:
done